### PR TITLE
Fix linting errors in command/Install.go

### PR DESCRIPTION
**Pull request by keydrevolta**
**What this PR does / why we need it**:
Fix openebs/openebs#780

### CODE REVIEW

File: command/install.go [hunk 1]
```diff
@@ -299,37 +286,33 @@ func (c *MayaAsNomadInstaller) installNomad() int {
...
-	return runop
+	return
 }
 
 //func (c *MayaAsNomadInstaller) setEtcd() int {
 
 //	var runop int
 
-//	c.Cmd = exec.Command("sh", SetEtcdScript, c.self_ip, c.self_ip_trim, c.etcd_cluster)
+//	c.Cmd = exec.Command("sh", SetEtcdScript, c.selfIp, c.selfIpTrim, c.etcdCluster)
```
Comment (prateekpandey14) [inline command/install.go]: here `selfIp --> selfIP` and `selfIpTrim --> selfIPTrim`

File: command/install.go [hunk 2]
```diff
@@ -338,28 +321,24 @@ func (c *MayaAsNomadInstaller) startNomadAsClient() int {
...
+	return
 }
 
-func (c *MayaAsNomadInstaller) setNomadAsClient() int {
-
-	var runop int
+func (c *MayaAsNomadInstaller) setNomadAsClient() (runop int) {
 
-	c.Cmd = exec.Command("sh", SetNomadAsClientScript, c.self_ip, c.fmt_master_ipnports)
+	c.Cmd = exec.Command("sh", SetNomadAsClientScript, c.selfIp, c.fmtMasterIpnports)
```
Comment (prateekpandey14) [inline command/install.go]: same as above comment.

File: command/install_openebs.go [hunk 3]
```diff
@@ -15,16 +15,16 @@ type InstallOpenEBSCommand struct {
 	Cmd *exec.Cmd
 
 	// all maya master ips, in a comma separated format
-	master_ips string
+	masterIps string
 
 	// self ip address
-	self_ip string
+	selfIp string
```
Comment (prateekpandey14) [inline command/install_openebs.go]: same as above

File: command/install.go [hunk 4]
```diff
@@ -30,7 +30,7 @@ type MayaAsNomadInstaller struct {
 	masterIps string
 
 	// all maya openebs ips, in a comma separated format
-	clientIps string
+	clientIPs string
```
Comment (prateekpandey14) [inline command/install.go]: these changes are not required. 
`clientIps` is right one here

File: command/install.go [hunk 5]
```diff
@@ -155,8 +155,8 @@ func (c *MayaAsNomadInstaller) validateIPs() (runop int) {
 // MayaAsNomadInstaller
 func (c *MayaAsNomadInstaller) initAsClient() (runop int) {
 
-	var masterIparr []string
-	var clientIparr []string
+	var masterIParr []string
+	var clientIParr []string
```
Comment (prateekpandey14) [inline command/install.go]: same as above , so remove this whole commit from the PR

**Comment by prateekpandey14** [comment]
@keydrevolta  unfortunately there was a same lint fix in another PR #149, which got merged first, 
so there are some conflicts , please resolve them.

**Comment by keydrevolta** [comment]
since I mixed all this branch, I'll leave the way it's and remake the PR, Thanks!

**Comment by prateekpandey14** [comment]
@keydrevolta  you have to resolve conflicts
1. Rebase with master 
2. resolve the conflicts 

helpful docs can we found [here](https://github.com/openebs/maya/blob/master/docs/developer.md)

**Comment by keydrevolta** [comment]
I rewrote the tree where I shouldnt

**Pull request closed without merging**